src/metricscollector/appmetricprune/prune/prune.go

```diff
@@ -0,0 +1,33 @@
+package prune
+
+import (
+	"db"
+	"time"
+
+	"code.cloudfoundry.org/lager"
+)
+
+const TokenTypeBearer = "bearer"
+
+type Prune struct {
+	logger   lager.Logger
+	metricDB db.MetricsDB
+}
+
+type ErrorResponse struct {
+	Code    string `json:"code"`
+	Message string `json:"message"`
+}
+
+func NewPrune(logger lager.Logger, metricDB db.MetricsDB) *Prune {
+	return &Prune{
+		logger:   logger,
+		metricDB: metricDB,
+	}
+}
+
+func (pr *Prune) PruneMetricsOlderThan(cutoffDays int) {
```

Comment:
yes, it doesn't change. You need to think about how this is really going to be used. I envision you create a Pruner that is just called on some time sequence. The cutoff is always the same.